k8s/vagrant/README.md

```diff
@@ -1,27 +1,36 @@
 # Installing Kubernetes Clusters on Ubuntu 16.04 using Vagrant
 
-OpenEBS provides vagrant boxes with prepackaged kuberentes images. There are different vagrant boxes created depending on the kubernetes release. The Vagrantfiles are organized here based on the kubernetes version used by the box. A deployer can choose the Vagrantfile, based on his choice of kubernetes version. 
+OpenEBS provides vagrant boxes with prepackaged kuberentes images. There are different vagrant boxes created depending on the kubernetes release. The Vagrantfiles are organized here based on the kubernetes version used by the box. 
 
-This Vagrantfile can be used on laptop or Baremetal server installed with Ubuntu 16.04 and Virtualization Enabled
+This Vagrantfile can be used on any machine with Virtualization Enabled, like laptop or baremetal server. 
 
+The below instructions will walk you through the following:
+- Verify Prerequisites
+- Download Vagrantfile
+- Bringup K8s Cluster
+- Install kubectl on the host 
+- Setup access to Kubernetes UI/Dashboard (Starting from Vagrantfile 1.7.5)
+- Setup OpenEBS
+- Launch demo pod
+
+*Note: The instructions are from an Ubuntu 16.06 host.*
 
 ## Prerequisites
 
 Verify that you have the following required software installed on your Ubuntu 16.04 machine:
```

Comment:
Verify that you have the following software installed on your Ubuntu 16.04 machine:

Reply:
done